plaid/api/institutions.py

```diff
@@ -36,18 +36,22 @@ def get_by_id(self, institution_id, _options=None):
             'options': options,
         })
 
-    def search(self, query, _options={}, products=None):
+    def search(self, query, products=None, account_filter=None, _options={}):
         '''
         Search all institutions by name.
 
-        :param  str      query:     Query against the full list of
-                                    institutions.
-        :param  [str]    products:  Filter FIs by available products. Optional.
+        :param  str      query:           Query against the full list of
+                                          institutions.
+        :param  [str]    products:        Filter FIs by available products.
+                                          Optional.
+        :param  dict     account_filter:  Filter FIs with specific account
```

Comment:
this should be better named as `account_subtypes` – similar to other filtering parameters in our endpoints. Or since there are 2 account filters:
```python
institutions.search(
  ...
  account_types = Array,
  account_subtypes = Array,
)
```

Reply:
Also – is this supported for all institutions? or will this only be available for Liabilities? I would consider putting this in the `options` object instead